specificatie/datum.yaml

```diff
@@ -40,12 +40,16 @@ components:
             datum:
               type: string
               format: date
+            langFormaat:
+              type: string
       required:
         - datum
+        - langFormaat
```

Comment:
door datum en langFormaat required te maken, wordt het onmogelijk met fields iets binnen de datum te vragen. Bijvoorbeeld fields=geboorte.datum.datum of fields=geboorte.datum.langFormaat leveren beide een schemafout op óf voldoen niet aan de fields feature.